var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -161,9 +161,16 @@ class Gromacs(CMakePackage, CudaPackage):
     variant(
         "nvshmem",
         default=False,
-        description="Enable nvshmem support for nvidia gpus",
+        when="@2023:+mpi+cuda",
```

Comment:
```suggestion
        when="@2024:+mpi+cuda",
```

Reply:
2024 is correct, I'm going to blame mobile on this one.